src/main/java/uk/gov/hmcts/reform/ccd/documentam/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -159,8 +159,8 @@ private boolean shouldSkip(final String caseId, final UUID documentId, final Doc
 
     private void verifyHashTokenValidity(DocumentHashToken documentHashToken,
                                          Document documentMetadata) {
-        String hashcodeFromStoredDocument =
-            generateHashToken(documentHashToken.getId(), documentMetadata, null, null);
+        String hashcodeFromStoredDocument = generateHashToken(documentHashToken.getId(), documentMetadata.getCaseId(),
+                              documentMetadata.getJurisdictionId(), documentMetadata.getCaseTypeId());
```

Comment:
These are in the wrong order with respect to the interface.

Reply:
changed on the interface